cloud/amazon/ec2_vol.py

```diff
@@ -332,7 +338,10 @@ def create_volume(module, ec2, zone):
     if volume is None:
         try:
             if boto_supports_volume_encryption():
-                volume = ec2.create_volume(volume_size, zone, snapshot, volume_type, iops, encrypted)
+                if kms_key_id:
```

Comment:
Thanks @defionscode for the feedback. That is indeed a good point. But is that consistent behaviour? On the one hand let the module fail, when a `kms_key_id` is given, and on the other gracefully ignore the `encryption` attribute, when the boto version is too old for it? Wouldn't it be more consistent to also ignore it?